docs/Deploying-Pega-on-EKS.md

```diff
@@ -0,0 +1,578 @@
+
+To setup your local system and save your AWS credentials and profile to the $USER/.aws file, enter:
+
+`$ aws configure`
+
+You are prompted for your AWS access credentials and details. Enter your own values. For guidance on completing each value, see [Quickly Configuring the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html#cli-quick-configuration).
+
+AWS Access Key ID [None]: your-key-ID
+AWS Secret Access Key [None]: your-secrete-access-key
+Default region name [None]: your-region-preference
+Default output format [None]:  specify your preference for a result format.
+
+With your credentials saved locally, you must push your Pega-provided Docker images to your Docker registry. For details on where the AWS CLI stores your credentials locally, see [Where Are Configuration Settings Stored?](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-where)
+
+### Managing SSL certificates in AWS certificate manager
+
+Pega supports the use of an HTTPS load balancer through a Kubernetes ingress, which requires you to configure the load balancer to present authentication certificates to the client. In EKS clusters, Pega requires that you use an AWS Load Balancer Controller (formerly named AWS ALB Ingress Controller). For an overview, see [Application load balancing on Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html).
+
+To configure this ingress controller, Pega allows your deployment to use the AWS Load Balancer Controller without a certificate for testing purposes; however, for the best security practice, Pega supports using an SSL certificate that you create or import into AWS Credential Manager. After you have your Amazon Resource Name (ARN) credential using this certificate, you must add an annotation to the web tier ingress configuration so the deployment passes your ARN certificate to the web tier.
+
+To import or create a new SSL certificate to support HTTPS, see [Importing Certificates into AWS Certificate Manager](https://docs.aws.amazon.com/acm/latest/userguide/import-certificate.html). After you have your ARN certificate, include a reference to the required ARN certificate using an appropriate "web tier" `ingress` annotation as in the example:
```

Comment:
It may be worth noting that if this annotation is omitted the ingress controller will attempt to automatically discover and attach certificates.  In my EKS lab I don't put on any annotation and it automatically associates the certificate I have in AWS Certificate Manager.

I think clients may want to use the annotation if they have multiple certificates or want to provide secondary certificates, etc.

Reply:
based on this feedback, changed this section to:
**TBD review following review comment from maracle6 EKS lab demos**
To configure this ingress controller, Pega allows your deployment to use the AWS Load Balancer Controller without a certificate for testing purposes; however, for the best security practice, it is a recommend practice to specify an SSL certificate that you create or import into AWS Credential Manager. After you have one your Amazon Resource Name (ARN) credential using this certificate or multiple certificates uploaded to AWS Credential Manager, use one of the following choices for the `annotation` parameter of the web tier ingress configuration:

- Leave it blank so that the deployment automatically associates the existing certificate with your ingress controller.
- Specify an ARN certificate out of multiple certificates or want to provide secondary certificates that the deployment associates with your ingress controller.